lib/plugins/inventory.js

```diff
@@ -528,7 +527,7 @@ function inject (bot, { hideErrors }) {
         mouseButton,
         action: actionId,
         mode,
-        item: Item.toNotch(click.item)
```

Comment:
Could you add a comment explaining this line ?

Reply:
in 1.8 it sends null as item with mode 2 or 4
i assumed this is up to 1.16.5 but that may not be correct, haven't tested

Reply:
Ok can you add a comment for this ?

Reply:
yes, did it

at first i thought you meant that i should explain it to you, my bad